langchain_benchmarks/tool_usage/agents.py

```diff
@@ -24,33 +25,44 @@ def _ensure_output_exists(inputs: dict) -> dict:
 
 class OpenAIAgentFactory:
     def __init__(
-        self, task: ToolUsageTask, *, model: str = "gpt-3.5-turbo-16k"
+        self,
+        task: ToolUsageTask,
+        *,
+        model: str = "gpt-3.5-turbo-16k",
```

Comment:
ooc: why do we need a default here/

Reply:
I can remove in a separate PR -- probably shouldn't be here